addons/web/static/src/search/search_model.js

```diff
@@ -1816,7 +1818,7 @@ export class SearchModel extends EventBus {
             }
         }
         orderBy = orderBy.length ? orderBy : this.globalOrderBy;
-        return typeof orderBy === "string" ? [orderBy] : orderBy;
+        return Array.isArray(orderBy) ? orderBy : [orderBy];
```

Comment:
return orderBy should work no? If not this means that globalOrderBy can be something different from what is expected by prop validation

Reply:
Yes, it would make sens. I updated the code :smile: